github/Repository.py

```diff
@@ -253,7 +253,9 @@
     from github.DependabotAlert import DependabotAlert
     from github.Deployment import Deployment
     from github.Download import Download
-    from github.EnvironmentDeploymentBranchPolicy import EnvironmentDeploymentBranchPolicyParams
+    from github.EnvironmentDeploymentBranchPolicy import (
+        EnvironmentDeploymentBranchPolicyParams,
+    )
```

Comment:
Is this intentional?

Reply:
Sorry this was an auto-format thing. Reverting it.